index.js

```diff
@@ -0,0 +1,353 @@
+import utils from './1sat_utils.cjs';
+
+
+let ISSUER_PRIVATE_KEY;
+let ISSUER_ADDRESS;
+let ISSUER_PUBLIC_KEY;
+const API_KEY = "<<YOUR TAAL API KEY HERE>>"
+let PROJECT_UID;
+let UTXO_LIST;
+let TX_OBJ;
+let TRANSACTION_UID;
+let OUTPUT_LIST = [];
+let TXID = '';
+let COLLECTION_ID='';
+let jsonResponse;
+let options;
+let faucet_needed = true;
+
+const STEPS_TO_TEST =['single', 'collection'];
+//const STEPS_TO_TEST =['1.1','1.2','1.3','1.4','1.5', '1.6', '1.7'];
+//const STEPS_TO_TEST =['2.1','2.2','2.3','2.4','2.5', '2.6', '2.7', '2.8', '2.9'];
```

Comment:
I think these comments should be removed or add a description about commented code.